lib/metrics/runtime.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const v8 = require('v8')
+const loopDelay = require('event-loop-delay')
+
+class RuntimeCollector {
+  constructor () {
+    this.stats = {
+      'node.active_handles': 0,
+      'node.active_requests': 0,
+      'node.cpu.system': 0,
+      'node.cpu.user': 0,
+      'node.event_loop.delay': 0,
+      'node.event_loop.times': 0,
+      'node.heap.allocated': 0,
+      'node.heap.used': 0
+    }
+
+    this.lastCpu = process.cpuUsage()
+    this.loopSampler = loopDelay()
+  }
+
+  start (interval) {
+    this.timer = setInterval(() => this.collect(), interval)
+    this.timer.unref()
+  }
+
+  stop () {
+    clearInterval(this.timer)
+  }
+
+  collect () {
+    // Handles and Requests
+    this.stats['node.active_handles'] = process._getActiveHandles().length
+    this.stats['node.active_requests'] = process._getActiveRequests().length
```

Comment:
Should we add a guard at these two lines in case the methods are removed/renamed in the future? There have been a few issues opened in core about changing these methods, though I think the risk is low. I'll leave it up to you :)

Reply:
fixed